api/test/config/loader.test.ts

```diff
@@ -1,3 +1,26 @@
-describe('test for test', () => {
-  expect('test').toBe('test');
+import { load } from '../../src/config/loader';
+import config from 'config';
+
+describe('Test config loader', () => {
+  it('should loading configurations from a test.yaml file is correct', () => {
+    expect(load()).toStrictEqual({
+      env: 'test',
+      http: { host: '127.0.0.1', port: 10100 }
+    });
+  });
+
+  it('should loading configurations from a test.yaml and does not overriden by env variable', () => {
+    const config = load();
+    process.env.env = 'production';
+    expect(config.env).not.toEqual('production');
```

Comment:
순서가 바뀌지 않았나요...?

Config 를 load() 해서 `config` 를 읽어들인 이후에 process env 를 바꾸는게 뭘 위한 테스트인 줄 모르습니다.
Process env 가 바뀌어져있을 때, config 필드가 덮어써지는지 안 되는지를 보려면 순서를 바꿔야 하는거 아니에요?

```suggestion
    process.env.env = 'production';
    const config = load();
    expect(config.env).not.toEqual('production');
```

Reply:
제가 잠깐 혼동한 것 같습니다. https://github.com/MovieReviewComment/Mr.C/pull/17/commits/886be2ecf6eb95767a225a2254bf4abb20e18b9a 에서내용 수정했습니다.